### ISSUE TITLE
`Project::refineSpatialCells` does refinement?

### ISSUE THREAD

**Issue by henry2004y** [fmihpc/vlasiator#642]
In the base class of Project, there is an implementation of `refineSpatialCells()` up to 2 levels. In the method comment, https://github.com/fmihpc/vlasiator/blob/b26df8f3e75f6d81c23e531849b4d16938794869/projects/project.cpp#L516

it says that this function does nothing and most likely should not be called. Do we need to keep the base method? Based on my quick scan the testAmr project https://github.com/fmihpc/vlasiator/blob/b26df8f3e75f6d81c23e531849b4d16938794869/projects/testAmr/testAmr.cpp#L277

basically has the same implementation.

**Comment by ykempf** [comment]
Yes we do as we use it e.g. in Flowthrough_amr for the testpackage.

**Comment by ykempf** [comment]
Well actually the comment is wrong, it doesn't do nothing, it refines for a certain width around a certain center: https://github.com/fmihpc/vlasiator/blob/b26df8f3e75f6d81c23e531849b4d16938794869/projects/project.cpp#L536.
